roles/etcdctl/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Create etcdctl script
+  template:
+    src: etcdctl.sh.j2
+    dest: "{{ bin_dir }}/etcdctl.sh"
+    mode: 0755
```

Comment:
I guess we only need that when using `etcd_kubeadm_enabled`, right?

Reply:
Yes. I only included this role protected with that condition since I did not want to touch the etcd role